org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/ProjectsManagerTest.java

```diff
@@ -30,9 +30,7 @@
 
 	@Test
 	public void testCreateDefaultProject() throws Exception {
-		TestVMType.setTestJREAsDefault();
```

Comment:
I know this is buggy but why was this removed?

Reply:
It's called in the `@Before` of the super class (`AbstractProjectsManagerBasedTest`)